src/jit/codegencommon.cpp

```diff
@@ -9406,9 +9422,6 @@ void                CodeGen::genFnEpilog(BasicBlock* block)
             compiler->unwindBegEpilog();
             unwindStarted = true;
         }
-        // mov FP into SP
-        inst_RV_RV(INS_mov, REG_SPBASE, REG_FPBASE);
-        compiler->unwindSetFrameReg(REG_FPBASE, 0);
```

Comment:
The epilog unwind codes are the inverse of the prolog unwind codes: you kind of need to think of them backwards. Ideally, the prolog and epilog codes are exact inverses, which allows for a smaller unwind codes encoding.

---
In reply to: [57753045](https://github.com/dotnet/coreclr/pull/3966#discussion_r57753045) [](ancestors = 57753045)